packages/labs/ssr/src/test/integration/tests/basic.ts

```diff
@@ -5512,6 +5513,167 @@ export const tests: {[name: string]: SSRTest} = {
       stableSelectors: ['le-internals-hydrate'],
     };
   },
+
+  /******************************************************
+   * ReactiveController tests
+   ******************************************************/
+
+  'ReactiveController: Constructor runs': () => {
+    return {
+      registerElements() {
+        class BasicController implements ReactiveController {
+          // To satisfy the ReactiveController interface.
+          declare hostConnected: undefined;
```

Comment:
interesting. i guess even if they're all optional, it expects at least one of the lifecycle methods to have been implemented to satisfy the interface, none of which gets called server side.
i guess this or a no-op method would be fine too.